### PR TITLE
Initial contribution for sdk js demo project

### PR DESCRIPTION
**Pull request by michelu89**

### CODE REVIEW

**Review by sha4be — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -1,2 +1,3 @@
-# sds-sdk-js-schematics-demo
-This repository is dedicated to SDS Working Group for developing an SDK for Aspect Model in Javascript
+# SDK JS Schematics :: Demo APP ✨
+
+To start the demo application and e2e test select the desired version, or read [Angular 14 DEMO APP](/angular14-demo-app/README.md).
```
Comment (atextor) [inline README.md]: I would have expected a little more context in the README, at least an explanation that it makes use of the esmf-sdk-js-schematics.

**Review by atextor — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (michelu89) [reply README.md]: Readme updated with new content

File: README.md [hunk 2]
```diff
@@ -1,3 +1,37 @@
...
+
+-   [Introduction](#introduction)
+-   [About](#about)
+-   [How it Works](#How-it-works)
+-   [Getting started](#getting-started)
+-   [License](#license)
+
+## Introduction
+
+This project serves as a demonstration of the ESMF Schematics Project in an Angular application.
```
Comment (atextor) [inline README.md]:
```suggestion
This project serves as a demonstration of the [ESMF Schematics Project](https://github.com/eclipse-esmf/esmf-sdk-js-schematics) in an Angular application.
```

File: README.md [hunk 3]
```diff
@@ -1,3 +1,37 @@
...
+## How it Works
+The ESMF Schematics project is leveraged here to generate a variety of tables in the Angular application. You can interact with these tables, exploring features such as sorting, filtering, and pagination. This demonstration highlights the practicality of ESMF Schematics in creating powerful, interactive elements in an Angular application.
+
+## Getting Started
+
+```bash
+git clone https://github.com/eclipse-esmf/esmf-sdk-js-schematics-demo.git
+```
+
+To start the demo application and e2e test select the desired version, or read [Angular 14 DEMO App](/angular14-demo-app/README.md).
```
Comment (atextor) [inline README.md]: What does "select the desired version" mean? What is someone supposed to run?

**Review by atextor — CHANGES_REQUESTED** [review]

File: README.md (hunk 3 above)
Comment (michelu89) [reply README.md]: Rewrite this sentence.

**Review by atextor — APPROVED** [review]

**Pull request merged**